### DISCUSSION TITLE
MACOS版本的ToolsFX没有CTF功能

### DISCUSSION THREAD

**Discussion by vul1ha0** [Leon406/ToolsFx#222] [Q&A] [answered]
![Snipaste_2024-01-25_13-26-01](https://github.com/Leon406/ToolsFx/assets/123882048/9296e2d8-7c28-47af-88a3-29c9706efa3b)

**Comment by Leon406** [comment, marked as answer]
https://github.com/Leon406/ToolsFx/blob/dev/README-zh.md#%E5%A6%82%E4%BD%95%E9%85%8D%E7%BD%AE

第一个参数配置下 true
